content/en/user-guide/integrations/crossplane/index.md

```diff
@@ -0,0 +1,226 @@
+aliases:
+  - /integrations/crossplane/
+---
+
+<img src="logo-crossplane.svg" width="500px" alt="Crossplane logo">
+
+## Overview
+
+[Crossplane](https://www.crossplane.io) is a cloud-native control plane framework, which offers an extensible backend that enables orchestrating applications and infrastructure via declarative APIs and resource definitions.
+
+Crossplane offers a native [AWS provider](https://github.com/upbound/provider-aws) which can be used to create and manage AWS cloud resources via the Crossplane platform.
+For example, it can be used to create S3 buckets, SQS queues, Lambda functions, among many other resources.
+At the time of writing, the Crossplane AWS provider supports a comprehensive set of some [900+ resource types](https://marketplace.upbound.io/providers/upbound/provider-aws).
+
+## Tutorial: Creating AWS resources in LocalStack via Crossplane
+
+In the following, we provide a step-by-step guide for installing Crossplane in a local test environment, and creating AWS resources (S3 bucket, SQS queue) in LocalStack via Crossplane.
+
+### Step 1: Installing Crossplane in local Kubernetes
+
+As a prerequisite, we need the following:
+* LocalStack running in local Docker
+* A local Kubernetes cluster:
+  * We can use the [embedded Kubernetes cluster](https://docs.docker.com/desktop/kubernetes) that ships with modern versions of Docker Desktop (can be easily enabled in the Docker settings)
+  * Alternatively, you can [create a local EKS cluster](https://docs.localstack.cloud/user-guide/aws/elastic-kubernetes-service/#create-an-embedded-kubernetes-cluster) in LocalStack directly, which will spin up a light-weight embedded `k3d` Kubernetes cluster in your Docker environment
+* The [`helm`](https://helm.sh) and [`kubectl`](https://kubernetes.io/docs/tasks/tools/#kubectl) command-line clients installed
+
+Once your `kubectl` is configured to point to the local Kubernetes cluster, we first install Crossplane via `helm`:
+{{<command>}}
+$ helm repo add crossplane-stable https://charts.crossplane.io/stable
+$ helm repo update
+$ helm install crossplane crossplane-stable/crossplane --namespace crossplane-system --create-namespace
+{{</command>}}
+
+The installation may take a few minutes. In parallel, we can install the `crossplane` command-line extensions for `kubectl`:
+{{<command>}}
+$ curl -sL https://raw.githubusercontent.com/crossplane/crossplane/master/install.sh | bash
+...
+$ sudo mv kubectl-crossplane /usr/local/bin
+{{</command>}}
+
+To confirm that the installation was successful, we can run these `kubectl` commands, which should yield output similar to the following:
+{{<command>}}
+$ kubectl crossplane --version
+v1.13.2
+
+$ kubectl get crds | grep crossplane
+compositions.apiextensions.crossplane.io                     2023-09-03T11:30:36Z
+configurations.pkg.crossplane.io                             2023-09-03T11:30:36Z
+...
+{{</command>}}
+
+### Step 2: Installing the Crossplane AWS Provider
```

Comment:
```suggestion
Installing the Crossplane AWS Provider
```

Reply:
Should we highlight these section titles, at least with bold font, or a `###` prefix?